### ISSUE TITLE
who are you

### ISSUE THREAD

**Issue by zequanhan** [ahm3draxa/ahm3draxa#1]
who are you？ Maybe we know each other

**Comment by ahm3draxa** [comment]
Hello, maybe we know each other of course. I am Ahmed Raza from Pakistan and you?

**Comment by zequanhan** [comment]
I am Kevin from China, and I think our research directions are very similar. So I think we might know each other?

**Comment by ahm3draxa** [comment]
yeah that's right. Nice to met you Kevin. I very keen learner. We can add each other for sure.

**Comment by zequanhan** [comment]
Okay, we can add each other and have in-depth communication on our project

**Comment by ahm3draxa** [comment]
sure.
